applications/ShapeOptimizationApplication/python_scripts/utilities/custom_sensitivity_heatmap.py

```diff
@@ -19,7 +19,7 @@
 
 # ==============================================================================
 # --------------------------------------------------------------------------
-class SensitivityHeatmapCalculator():
+class SensitivityHeatmapCalculator:
```

Comment:
From ChatGPT:
Do I need to use object while creating class in python?

No, you don't need to use "object" while creating a class in Python. However, by default, all classes in Python are derived from the "object" class unless specified otherwise. So, including "object" in the class definition is optional but common practice. It helps to make sure that the class has all the default properties and methods of a Python object.